README.md

```diff
@@ -1,17 +1,30 @@
 # GlotPress
 
-This is a community-backed plugin which brings [GlotPress](https://github.com/GlotPress/GlotPress) into WordPress as a plugin.
+GlotPress is a plugin for standalone installation of WordPress to help translators collaborate.
```

Comment:
"for standalone installation of WordPress", that sounds a bit odd.

What about "GlotPress is a plugin to set up your own collaborative, web-based software translation tool."?